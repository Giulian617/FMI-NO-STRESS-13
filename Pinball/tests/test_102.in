100 1
21 1 2
98 1 4
1 41 7
50 1 9
1 58 8
1 7 3
1 15 1
72 1 2
65 1 3
9 1 3
1 54 2
47 1 8
95 1 8
1 60 8
57 1 10
1 30 4
1 93 5
97 1 10
1 17 10
1 32 8
3 1 9
1 10 4
42 1 6
18 1 5
1 52 7
1 71 9
25 1 9
68 1 1
99 1 2
29 1 8
5 1 3
1 96 2
1 66 5
62 1 7
80 1 1
1 85 9
53 1 6
87 1 8
1 90 10
1 70 10
1 22 4
1 61 3
1 36 2
73 1 8
76 1 8
1 14 9
35 1 8
77 1 8
1 6 5
1 11 7
86 1 1
37 1 4
1 67 8
1 92 4
38 1 8
89 1 6
1 78 9
1 13 9
1 88 10
19 1 6
1 83 4
69 1 6
1 63 9
1 26 1
39 1 9
84 1 3
1 24 5
1 49 5
81 1 1
1 33 10
1 4 7
28 1 3
2 1 10
1 75 1
74 1 8
1 34 2
1 94 6
59 1 2
1 79 6
16 1 1
1 45 3
1 44 5
51 1 5
8 1 4
1 64 2
1 40 1
55 1 10
1 48 7
1 46 1
43 1 3
82 1 7
23 1 3
12 1 8
20 1 5
1 91 10
100 1 2
1 27 4
56 1 9
1 31 2
