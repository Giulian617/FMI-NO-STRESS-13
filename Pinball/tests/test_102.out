400538935
218575307
100040095
424772431
933446201
424772431
988029083
424772431
988029083
100040095
395623336
218575307
218575307
153824012
153824012
400538935
933446201
860925949
933446201
622720753
988029083
424772431
933446201
218575307
153824012
988029083
424772431
395623336
988029083
622720753
395623336
400538935
622720753
395623336
622720753
988029083
395623336
218575307
153824012
100040095
860925949
424772431
933446201
424772431
153824012
395623336
100040095
933446201
218575307
933446201
100040095
860925949
622720753
400538935
218575307
400538935
395623336
622720753
395623336
424772431
100040095
218575307
622720753
424772431
933446201
395623336
153824012
860925949
400538935
218575307
622720753
395623336
395623336
153824012
395623336
395623336
218575307
860925949
153824012
153824012
100040095
988029083
424772431
218575307
153824012
395623336
400538935
860925949
400538935
400538935
988029083
933446201
860925949
395623336
622720753
400538935
988029083
622720753
622720753
